tests/integration/test_aws/data/test_cases/remove_from_bucket_test_module/cases_remove_from_bucket.yaml

```diff
@@ -1,8 +1,17 @@
 - name: cloudtrail_remove_from_bucket
   description: CloudTrail remove from bucket configurations
   configuration_parameters:
-    BUCKET_TYPE: cloudtrail
-    BUCKET_NAME: wazuh-cloudtrail-integration-tests
+    BUCKET_TYPE: vpcflow
+    BUCKET_NAME: wazuh-vpcflow-integration-tests
   metadata:
-    bucket_type: cloudtrail
-    bucket_name: wazuh-cloudtrail-integration-tests
+    bucket_type: vpcflow
+    bucket_name: wazuh-vpcflow-integration-tests
```

Comment:
Are these changes correct? Cloudtrail will not be tested with its own bucket.

Reply:
Fixed in [61c7a24](https://github.com/wazuh/wazuh-qa/pull/3699/commits/61c7a24f884d3b3ec1d55f63bb7d9ff6853456f3)